pajbot/apiwrappers.py

```diff
@@ -248,7 +249,7 @@ def __init__(self, client_id=None, oauth=None, strict=True):
         APIBase.__init__(self, strict)
 
         self.base_url = "https://api.twitch.tv/api/"
-        self.kraken_url = "http://127.0.0.1:7221/kraken/"
+        self.kraken_url = f"http://{os.environ.get('KRAKEN_URL', '127.0.0.1:7221')}/kraken/"
```

Comment:
We currently want to remain compatible with Python 3.5 (because the documented/recommended deployment is Debian stable), which doesn't support the `f""` format syntax. To remain compatible this needs to be changed to classic style format.